Automattic/sensei
# Use Fill button style when Complete Lesson button is the only action available

Resolves #6924.

## Proposed Changes
Use the button Fill style for the _Complete Lesson_ button when it's the only action on the page.

## Testing Instructions


1. Open a lesson without a quiz in Learning Mode.
2. Ensure the _Complete Lesson_ button uses the Fill button style.
3. Open a lesson with a quiz in Learning Mode.
4. Ensure the _Complete Lesson_ button uses the Outline button style and that the _Take Quiz_ button uses the Fill button style.

## Pre-Merge Checklist
